OracleWebLogic/samples/12212-msiserver/README.md

```diff
@@ -18,21 +18,14 @@ Next, to build the base msi image, run:
 
 Finally, to start the Managed Server in MSI mode, run:
 
-        $ docker run -d -p 8011:8011 12212-msiserver
+        $ docker run -p 8011:8011 12212-msiserver

```

Comment:
Hi @Djelibeybi I removed -d so that stdout can be shown on the console for further investigation. You had mentioned earlier that "exec bash" wasn't a best practice. Anyways, I'll put it back and user "docker logs" as you suggested. Thanks.